src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/out/msg/ErrorMsg.java

```diff
@@ -10,25 +10,40 @@
 
     public final String id;
     public final Long eventId;
+
+    @JsonProperty("zip_file_name")
     public final String zipFileName;
+
+    @JsonProperty("jurisdiction")
     public final String jurisdiction;
+
+    @JsonProperty("po_box")
     public final String poBox;
+
+    @JsonProperty("document_control_number")
     public final String documentControlNumber;
+
+    @JsonProperty("error_code")
     public final ErrorCode errorCode;
+
+    @JsonProperty("error_description")
     public final String errorDescription;
     public final String service;
 
     // region constructors
     @SuppressWarnings("squid:S00107") // number of params
     public ErrorMsg(
-        @JsonProperty(value = "id", required = true) String id,
+        // TODO: make id transient when the application stops sending error notifications
+        @JsonProperty(value = "id", required = true)
+        String id,
+        // TODO: remove eventId when the application stops sending error notifications
         @JsonProperty(value = "eventId", required = true) Long eventId,
-        @JsonProperty(value = "zipFileName", required = true) String zipFileName,
+        @JsonProperty(value = "zip_file_name", required = true) String zipFileName,
         @JsonProperty("jurisdiction") String jurisdiction,
-        @JsonProperty("poBox") String poBox,
-        @JsonProperty("documentControlNumber") String documentControlNumber,
-        @JsonProperty(value = "errorCode", required = true) ErrorCode errorCode,
-        @JsonProperty(value = "errorDescription", required = true) String errorDescription,
+        @JsonProperty("po_box") String poBox,
+        @JsonProperty("document_control_number") String documentControlNumber,
+        @JsonProperty(value = "error_code", required = true) ErrorCode errorCode,
+        @JsonProperty(value = "error_description", required = true) String errorDescription,
         @JsonProperty(value = "service", required = true) String service
```

Comment:
if this is an outgoing model, one that we serialise *to* json, than those annotations don't do anything.

Reply:
It will be an outgoing model after the switch is complete, but for now this model represents both outgoing and incoming message. The processor sends this message, but also reads and processes it.